code/modules/cm_preds/yaut_bracers.dm

```diff
@@ -752,7 +748,12 @@
 		log_game("[key_name_admin(usr)] has enabled their cloaking device.")
 		if(!silent)
 			M.visible_message(SPAN_WARNING("[M] vanishes into thin air!"), SPAN_NOTICE("You are now invisible to normal detection."))
-			playsound(M.loc,'sound/effects/pred_cloakon.ogg', 15, 1)
+			var/sound_to_use
+			if(invisibility_sound == "Modern")
+				sound_to_use = 'sound/effects/pred_cloakon_modern.ogg'
+			else
+				sound_to_use = 'sound/effects/pred_cloakon.ogg'
+			playsound(M.loc, sound_to_use, 15, 1, 10)
```

Comment:
I'm not sure why you're changing the radius of the sound cue, certainly not to beyond a screen size.
For something very stealthy that's a lot of range on the noise. Also further than scout cloak (3.75 tiles).